reventlov/bot.py

```diff
@@ -1,10 +1,11 @@
+from future.utils import iteritems
```

Comment:
This is to enable Python 2/3 compatibility.

Reply:
Why Python 2 support?